src/opnsense/mvc/app/models/OPNsense/OpenVPN/OpenVPN.xml

```diff
@@ -274,11 +274,12 @@
                     <OptionValues>
                         <client-to-client>client-to-client</client-to-client>
                         <duplicate-cn>duplicate-cn</duplicate-cn>
+                        <float>float</float>
                         <passtos>passtos</passtos>
                         <persist-remote-ip>persist-remote-ip</persist-remote-ip>
-                        <route-nopull>route-nopull</route-nopull>
-                        <route-noexec>route-noexec</route-noexec>
                         <remote-random>remote-random</remote-random>
+                        <route-noexec>route-noexec</route-noexec>
+                        <route-nopull>route-nopull</route-nopull>
```

Comment:
why move these options around? adding `float` is fine, but let's try to keep the diff as small as possible unless there is a good reason to move things.

Reply:
sorting them alphabetically can help users when these lists grow beyond a certain size. certainly worth having (and consistent with out approach)

Reply:
I agree with both comments. Despite the "sort" commit not matching the PR expectations, I've favored consistency afterwards over as small as possible as the diff itself is still small and less than a page.

Reply:
It's also two commits. Using rebase option here on GitHub should make that crystal clear in the code history.